flow/src/main/java/com/sequenceiq/flow/controller/FlowController.java

```diff
@@ -65,4 +67,16 @@ public FlowCheckResponse hasFlowRunningByChainId(String chainId) {
     public FlowCheckResponse hasFlowRunningByFlowId(String flowId) {
         return flowService.getFlowState(flowId);
     }
+
+    @Override
+    @AccountIdNotNeeded
+    public Page<FlowLogResponse> getFlowLogsByFlowIds(List<String> flowIds, int size, int page) {
+        return flowService.getFlowLogsByIds(flowIds, PageRequest.of(page, size));
+    }
+
+    @Override
+    @AccountIdNotNeeded
+    public Page<FlowCheckResponse> getFlowChainsStatusesByChainIds(List<String> chainIds, int size, int page) {
+        return flowService.getFlowChainsByChainIds(chainIds, PageRequest.of(page, size));
+    }
```

Comment:
tests are missing for this . We should use DataJpaTest which can also verify the paging.